pages/content/migration-guidance.mdx

```diff
@@ -6,9 +6,10 @@ title: "Migrating from GOV.UK PaaS"
 
 ## Last updated
 
-- 18 May 2023 - added new guidance ['Finalise your GOV.UK PaaS account'](#finalise-your-govuk-paas-account) 
-- 5 Jan 2023 - added new guidance ['PaaS CDB Migration'](#paas-cdn-migration) 
-- 2 November 2022 - first published
+- 2023-08-15 - new guidance section ['Publishing static pages to GitHub Pages'](#publishing-static-pages-to-github-pages)
```

Comment:
let's use a human-readible date format as before

Reply:
as a human I prefer [ISO8601 https://en.wikipedia.org/wiki/ISO_8601 format dates :) however I will change the dates